lib/delayed_paperclip/railtie.rb

```diff
@@ -1,26 +1,21 @@
-require 'paperclip'
-require 'delayed_paperclip'
+require "paperclip"
+require "delayed_paperclip"
 
 module DelayedPaperclip
-  if defined? Rails::Railtie
```

Comment:
trying to remember why this was added in the first place - I don't think it'll be a significant problem if it is an issue